docs/interactions/structured-messages/carousel.md

```diff
@@ -120,6 +120,10 @@ Primary parameters are used by default, however, some parameters are unique or o
 |-|-|
 | **`structured_content.items.attachment_id`** | Supports gif, jpg, jpeg, png formats. Supports private attachments. [Upload attachments](../../../basics/uploads) for you own custom images.<br>On Engage Messaging Web, if the width of the **first** image is bigger than the height, every image will be displayed with a 5:3 ratio. Otherwise, a 1:1 ratio will be used.<br>Minimal recommended size with a 1:1 ratio: **258x258**<br>Minimal recommended size with a 5:3 ratio: **258x155** |
 | **`structured_content.items.url`** | Deep links are supported. |
+| **`structured_content.items.target`** | **Optional**. **Ignored** when the `url` is empty. Behavior applied when clicking on the url. Can be `webview` to open url on a [webview](../structured-messages/webview) above the chat, `open` to open url in new tab, or  `current` to open url in current tab. Defaults to `open` when not specified.|
```

Comment:
Ugh, can you please remove the double space everywhere it's present? I see multiple of them in this PR diff when doing ctrl+f

```suggestion
| **`structured_content.items.target`** | **Optional**. **Ignored** when the `url` is empty. Behavior applied when clicking on the url. Can be `webview` to open url on a [webview](../structured-messages/webview) above the chat, `open` to open url in new tab, or `current` to open url in current tab. Defaults to `open` when not specified.|
```

Reply:
Should be better after 90a0b3c, sorry for that